pkgs/by-name/lu/lutgen-studio/package.nix

```
@@ -0,0 +1,71 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  makeWrapper,
+
+  fontconfig,
+  libGL,
+  libxkbcommon,
+  openssl,
+  wayland,
+  xorg,
+
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lutgen-studio";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "ozwaldorf";
+    repo = "lutgen-rs";
+    tag = "${pname}-v${version}";
```

we shouldn't be reusing pname its bad practice.

```suggestion
    tag = "lutgen-studio-v${version}";
```